packages/web/src/components/ui/Card/Card.module.scss

```diff
@@ -0,0 +1,231 @@
+@import '~styles/_vars.scss';
+@import '~styles/responsive.scss';
+
+.card {
+  position: relative;
+  background: #ffffff;
+  border-radius: 24px;
+  padding: 24px;
+  box-shadow: $shadow;
+  width: 100%;
+  max-width: 600px;
+  min-height: 248px;
+  display: flex;
+  flex-direction: column;
+
+  @include lg {
+    padding: 20px;
+  }
+
+  @include mobile-and-tablet {
+    min-height: 204px;
+    box-shadow: $shadow-small;
+  }
+}
+
+.header {
+  display: flex;
+  align-items: center;
+}
+
+.icon {
+  flex-shrink: 0;
+  margin-right: 16px;
+  display: inline-flex;
+
+  img {
+    width: 100%;
+    max-width: 52px;
+    object-fit: cover;
+  }
+}
+
+.title {
+  font-size: 26px;
+  font-weight: 500;
+  line-height: 1.54;
+
+  @include mobile-and-tablet {
+    font-size: 20px;
+    line-height: 1.4;
+  }
+}
+
+.description {
+  margin-top: 4px;
+  text-overflow: ellipsis;
+  overflow: hidden;
+  white-space: nowrap;
+  color: $gray-text;
+}
+
+.tagsWrapper {
+  margin-top: 28px;
+  flex: 1;
+  display: flex;
+  flex-direction: column;
+
+  @include mobile-and-tablet {
+    margin-top: 20px;
+  }
+}
+
+.tags {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: flex-start;
+
+  & > * {
```

Comment:
Let's avoid `*` selectors as well. Since CSS selectors are read by a browser from right to left, this selector will be checked for any html element on a page.

Reply:
This is potentially true for this case, but I'd advise against banning usage of `*` selectors, we'll likely to end up with a lot of unneeded complexity in react-side of markup.

Reply:
I can't think about any useful `*` use case other than a single usage in a `global.css` for things like `box-sizing`